web/app.js

```diff
@@ -1215,6 +1230,26 @@ var PDFViewerApplication = {
       return;
     }
     this.pdfPresentationMode.mouseScroll(delta);
+  },
+
+  bindEvents: function pdfViewBindEvents() {
+    var eventBus = this.eventBus;
+
+    eventBus.on('resize', webViewerResize);
+    eventBus.on('localized', webViewerLocalized);
+    eventBus.on('fileinputchange', webViewerFileInputChange);
```

Comment:
The function that is referenced here is placed within a `GENERIC` preprocessor tag, shouldn't this line be too, in order to prevent any possible issues?